tests/DevApps/aspnet-mvc/OwinWebApi/OwinWebApi.csproj

```diff
@@ -23,7 +23,6 @@
     <IISExpressWindowsAuthentication />
     <IISExpressUseClassicPipelineMode />
     <UseGlobalApplicationHostFile />
-    <LangVersion>8.0</LangVersion>
```

Comment:
Please add the `<LangVersion>` in https://github.com/AzureAD/microsoft-identity-web/blob/master/tests/DevApps/aspnet-mvc/Directory.Build.props.
This will do for both the OwinWebApi and OwinWebApp.

Reply:
I added the <LangVersion> in https://github.com/AzureAD/microsoft-identity-web/blob/master/tests/DevApps/aspnet-mvc/Directory.Build.props. However, I get the following error after building 

<img width="452" alt="error_msg" src="https://github.com/AzureAD/microsoft-identity-web/assets/38921563/cfa69340-7c82-49b2-bf6b-7a237449922a">

Reply:
your branch works.